.codesandbox/ci.json

```diff
@@ -39,6 +39,7 @@
     "/examples/material-cra",
     "/examples/material-cra-ts",
     "/examples/joy-cra-ts",
+    "/examples/base-cra",
```

Comment:
I'm not sure if we want to have sandboxes for more examples - perhaps just the code in our repo would be sufficient?

Reply:
Cool, I can drop these. Wasn't sure if I should follow the pattern with the other examples or not.